packages/@cdktf/hcl2cdk/test/hcl2cdk.test.ts

```diff
@@ -31,9 +31,12 @@ const cdktfBin = path.join(__dirname, "../../../cdktf-cli/bin/cdktf");
 const cdktfDist = path.join(__dirname, "../../../../dist");
 
 const createTestCase =
-  (describeFn: typeof describe) =>
+  (opts: { skip?: true }) =>
   (name: string, hcl: string, shouldSynth: Synth) => {
-    describeFn(name, () => {
+    if (opts.skip) {
+      return;
```

Comment:
We could use this:
```suggestion
      describe.skip(name, () => {});
      return;
```
to let jest know about all skipped tests (it would print them in that case). No hard feelings though.